packages/demo/package.json

```diff
@@ -64,6 +64,9 @@
     "@web3-onboard/xdefi": "^2.0.5",
     "@web3-onboard/zeal": "^2.0.3",
     "@web3-onboard/particle-network": "^2.0.1",
+    "@web3-onboard/passport": "2.0.0",
+    "@0xpass/webauthn-signer": "2.0.0",
+    "@0xpass/passport": "2.1.1",
```

Comment:
Are these deps required?

Reply:
Yes these deps are required

Reply:
Why are they required if they are part of the `@web3-onboard/passport` package?

Reply:
actually for this you're right passport can be removed but webauthn-signer is a required dependency.

Reply:
@k-xo if it is required for the package to work should it be added to the package dependencies?
If a dependency is required for the package to work and isnt included it should be documented

Reply:
We have multiple dependency types, and support a specific signer interface, so this setup is setting up the flow where users are using passkeys / webauthn signer, so it is required in the application context of setting up the signer, and is the most convenient way of setting it up, however it is not required in the sense it should be included in the onboard package. 

so in the demo we have the passport onboard package and the webauthn signer, but in the actual passport package we have the core passport sdk, as well as our viem client.